### ISSUE TITLE
Create a Table of Contents for README.md

### ISSUE THREAD

**Issue by ConsenSys-Academy** [ConsenSys-Academy/Blockchain-Developer-Bootcamp#225]
To allow people to easily browse and acquaint themselves with the technical content, it would be helpful to have a Table of Contents on the README.md page. This Table of Contents should shows the overall structure of the course, with the modules and lessons linked.

Since the overall nested structure is really large, for readability, it might be helpful to use something like the `<details>` and `<summary>` html features when building this Table of Contents. [More on those html features here.](https://gist.github.com/scmx/eca72d44afee0113ceb0349dd54a84a2)

**Comment by yeahmat** [comment]
to confirm my understanding , we'd want something like:


## Table of contents 

<details>
   <summary>
      <a href="https://github.com/ConsenSys-Academy/Blockchain-Developer-Bootcamp/tree/staging/docs/S00-intro" >S00-intro</a>
  </summary>

  <ul>
    <li><a href="https://github.com/ConsenSys-Academy/Blockchain-Developer-Bootcamp/blob/staging/docs/S00-intro/L1-course-intro/index.md">L1-course-intro</a></li>
...
 </ul>
</details>

**Comment by yeahmat** [comment]
@ConsenSys-Academy Is there a preference on relative vs absolute pathing for the links?

**Comment by ConsenSys-Academy** [comment]
@yeahmat Hey there! That's exactly right, the only note I'd have would be to use the title of the section / module / lesson as the link text. So `(S00-intro)[link here]` would be something like `(Introduction to the Course)[docs/S00-intro]`. Does that make sense? I can help with this if needed.

No preference on relative vs absolute pathing for the repo, I don't think.

Some sections are also missing content (their available as a video on the LMS that we'll need to migrate over), it might be good to note that with an asterisk, if possible.

Once you start working on this, though, we can collab!

**Comment by yeahmat** [comment]
> So (S00-intro)[link here] would be something like (Introduction to the Course)[docs/S00-intro]. Does that make sense?

Absolutely, grab the H1 text from the actual document, and not the document name itself. 👍 

> No preference on relative vs absolute pathing for the repo, I don't think. 

I think relative would be easier manage in the long term. I'll go relative unless there are objections 🙂 

> Some sections are also missing content (their available as a video on the LMS that we'll need to migrate over), it might be good to note that with an asterisk, if possible.

Confirmed, if no content exists in the Markdown file for the section, leave an asterisk. 

---
I'll get started a bit later this afternoon. Mostly copy/pasting so it should be able to have it in a review state by 05/13

**Comment by ConsenSys-Academy** [comment]
@yeahmat awesome! let me know if you have any more questions. Sometimes we miss pings on here so feel to ping in the Edu-DAO discord as well!

**Comment by ConsenSys-Academy** [comment]
@yeahhhmat Wanted to check in ... Any progress on this?

**Comment by ConsenSys-Academy** [comment]
@yeahhhmat One more ping!

**Comment by lonasozo** [comment]
Hey guys, I've started to create the table contents for myself, I can create a pull request so we can work together

**Comment by lonasozo** [comment]
Guys I think this should be closed (?)
